default/stringtables/en.txt

```diff
@@ -4219,6 +4219,39 @@ Next are icons showing the player empire total [[encyclopedia INDUSTRY_TITLE]],
 
 On the right hand side of the control bar are a number of icons for toggling display of other significant UI windows. Tooltips are available for these.'''
 
+DESIGN_WINDOW_ARTICLE_TITLE
+Design Window
+DESIGN_WINDOW_ARTICLE_TEXT
+'''The design window is for viewing or altering the designs of monsters or ships.
+
+There are four sub windows:  Design Starts, Ship Parts, Design Details and the Pedia.
+
+
+Design Starts has four tabs: empty Hulls, Finished designs, Saved designs from previous games and Monster designs for inspection.  Drag or double click hulls, designs or monsters to edit/examine them on the workbench.
```

Comment:
I've capitalized words that are tab names from the UI, or context menu commands.  I could remove the capitalization or use quotes.